src/bootman/kernel.c

```diff
@@ -206,7 +206,19 @@ Kernel *boot_manager_inspect_kernel(BootManager *self, char *path)
                 buf = NULL;
         }
 
+        /* Use kernel's command line and /etc/kernel/cmdline if present */
+        if (self->cmdline) {
+                char *cm = NULL;
+                if (asprintf(&cm, "%s %s", kern->cmdline, self->cmdline) < 0) {
```

Comment:
So I am not remembering and I failed search so how do conflicting (different root= or whatever) get resolved in the kernel commandline? Only commenting to ensure having the system options first is the correct way to get user configured is taken.